xenon.yml.example

```diff
@@ -1,7 +0,0 @@
-browsers:
-  - name: chrome
-    driver_path: /usr/local/bin/chromedriver
-    max_sessions: 10
-  - name: firefox
-    driver_path: /usr/local/bin/geckodriver
-    max_sessions: 10
```

Comment:
Why not leave the example file there and add ports to make it work?

That way new users have something they can copy to get up and running quickly.